elastic/kibana
# [ResponseOps][Alerting] ES query rule DSL editor doesn't update when you copy/paste a query

Resolves https://github.com/elastic/kibana/issues/161595


## Summary

Captures changes in the editor when copying/pasting in values.


### To verify

- Create a new Es Query rule using DSL
- Paste runtime mappings in your query and don't type anything else in the editor
- Verify that the runtime fields are listed in the group by or aggregate over options below the query
